ksp-processors/src/main/kotlin/generation/shared/Names.kt

```diff
@@ -1,16 +1,24 @@
+@file:OptIn(ExperimentalContracts::class, DelicateKotlinPoetApi::class)
```

Comment:
i don't like the opt-ins here, why not keep the old approach of using strings?

Reply:
Because this one is actually type-safe and I like it